docs/proposals/images/node-os-image-cache.plantuml

```diff
@@ -0,0 +1,25 @@
+@startuml
```

Comment:
nit: There's a Makefile in `docs` that generates the images for you with a specific (but old) version of PlantUML that might format things a bit differently but more consistently with the diagrams in other proposals:

```
touch docs/proposals/images/node-os-image-cache.plantuml && make -C docs diagrams
```